hooks/pre-commit

```diff
@@ -47,7 +47,7 @@ def get_script_version(file_name):
 
 # check and get script directory changes
 subprocess.check_output(
-    'git fetch https://github.com/weecology/retriever.git master', shell=True)
+    'git fetch https://github.com/harshitbansal05/retriever.git development', shell=True)
```

Comment:
Do we want to keep this here `harshitbansal05`

Reply:
Right now, I kept this only for testing. Later, we can change it to retriever repository.

Reply:
I think it's better if we change it to the retriever-recipes repository right now instead of your temporary git. We can wait for [retriever-recipes PR 2](https://github.com/weecology/retriever-recipes/pull/2) to go in first before merging this in.

Reply:
Ok, I would change it as soon as recipes pr is merged.

Reply:
alright, recipes pr should be merged

Reply:
@zhangcandrew the above url refers to the retriever repository, which has not been merged. I have updated the url for the recipes scripts repository.

Reply:
ahh ok I see what you mean by this then. I see you have 6 commits, let's split this PR into two PR's one with just the changes up to 78f5d64eeb09a9194dc5a75cadba3d029a9ee702, and we can have another with commits after. That way we can get the first half in and then commit the second half successfully without having to reference an intermediary repo like this.